backend/matching-service/src/types/UserQueueRequestDto.ts

```diff
@@ -15,16 +19,17 @@ export class UserQueueRequestDto {
     @IsNotEmpty()
     userId: string
 
-    constructor(proficiency: Proficiency, topic: Category, userId: string) {
+    constructor(proficiency: Proficiency, difficulty: Complexity, topic: Category, userId: string) {
```

Comment:
maybe would be better to keep the naming same? like complexity: Complexity etc. so less confusion

Reply:
i think using difficulty would help frontend understand what input to pass in for the api call, using the complexity enum

Reply:
i think lets just standardize it as complexity (lol didnt see the second comment and i alr renamed everything haha)